dsnp/index.ts

```diff
@@ -29,33 +31,40 @@ export {
 type PayloadLocation = "IPFS" | "OnChain" | "Itemized" | "Paginated";
 type Settings = "AppendOnly" | "SignatureRequired";
 
+// Note: Schema should be retained for versions with published data on mainnet
+export type DSNPVersion = "1.1" | "1.2" | "1.3";
```

Comment:
nit: It's worth mentioning that these versions are different and independent of the SchemaVersion stored on chain for each name.

Reply:
Yeah, that confused me for a minute until I realized they're DSNP versions, nothing to do with the schema version associated with the schema name on-chain.

Reply:
Good point, I'll add more commentary.